docs/Deployment.md

```diff
@@ -267,6 +267,18 @@ This value should be sufficiently greater than `raft-interval` and
 
 *Default*: 1000
 
+### `connection-timeout`
+
+the number of milliseconds the cluster will wait for connections to other nodes to succeed before timing out.  We reccomend it be set if modifying `election-timeout` to be at least a 3x multiple of it.
```

Comment:
```suggestion
The number of milliseconds the cluster will wait for connections to other nodes to succeed before timing out.  We recommend it be set if modifying `election-timeout` to be at least a 3x multiple of it.
```

Typo fix above. What is the rationale behind connection-timeout >= 3 * election-timeout?